php/libraries/NDB_Form.class.inc

```diff
@@ -54,7 +54,7 @@ class NDB_Form extends NDB_Page
      * @param string $page       Identifies the page to show
      * @param string $identifier Variables passed to form
      *
-     * @return void
+     * @return object
```

Comment:
```suggestion
     * @return \NDB_Form
```

I think the factory for an NDB_Form should always return some type of NDB_Form, not a generic object.